modules/special-rules/pages/unbreakable.adoc

```diff
@@ -0,0 +1,20 @@
+= Unbreakable
+
+Some troops have such high morale, or lack of individual willpower, that they never break.
+
+[NOTE.e40k]
+====
+In comparison to _Epic 40,000_, we've separated this ability from part of the Tyranid Synapse Creatures rules.
+====
+
+---
+
+Where your detachment has the {unbreakable} ability:
+
+* The detachment can never become Broken.
+
+{blank}[TODO: What about the Retreat move? The "can never be broken" aspect of the original Tyranid Synapse Creatures rule includes text about this: link:https://thehobby.zone/resources/e40k-compendium/Content/Armies/ArmiesOfTheTyranids/HiveFleetArmyList.htm[Hive Fleet army list^] (external URL). And this is different to link:https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/WarEngines/NoRetreatNoSurrender.htm[No retreat, no surrender] for war engines.]
```

Comment:
See my Synapse suggestion, which could go here instead:

```suggestion
NOTE: If you lose a close combat or firefight with the detachment then it must still make a Retreat move, unless it also has the (no-retreat} ability.
```

Reply:
Adding the Retreat move note here, as it looks like a more sensible place than in Synapse.